patches.def.h

```diff
@@ -347,6 +347,12 @@
  */
 #define BAR_PADDING_PATCH 0
 
+/* Same as barpadding patch but specifically written for the vanitygaps patch. It adds the same
+ * outer horizontal and vertical padding as vanity gaps. Moreover, the padding is toggled whenever
+ * vanitygaps are toggled.
+ */
+#define BAR_GAPS_PADDING_PATCH 0
```

Comment:
The name sounds a bit odd, I may change this to BAR_PADDING_GAPS_PATCH instead unless I come up with a better name for this.

Reply:
I'm fine with whatever name you come up with. :)